spec/models/tslr_claim_spec.rb

```diff
@@ -55,12 +55,21 @@
     end
 
     context "that has bank details" do
-      it "validates the length of account_number and account_sort_code" do
-        expect(TslrClaim.new(account_number: "123456789")).not_to be_valid
-        expect(TslrClaim.new(account_sort_code: "10111213")).not_to be_valid
-
-        expect(TslrClaim.new(account_number: "12345678")).to be_valid
-        expect(TslrClaim.new(account_sort_code: "101112")).to be_valid
+      it "validates the format of bank_account_number" do
+        expect(TslrClaim.new(bank_account_number: "ABC12 34 56 789")).not_to be_valid
+        expect(TslrClaim.new(bank_account_number: "12-34-56-78")).to be_valid
+      end
+      it "validates the length of bank_account_number" do
+        expect(TslrClaim.new(bank_account_number: "12 34 56 78 90")).not_to be_valid
+        expect(TslrClaim.new(bank_account_number: "12-34-56-78")).to be_valid
+      end
+      it "validates the format of bank_sort_code" do
+        expect(TslrClaim.new(bank_sort_code: "ABC12 34 567")).not_to be_valid
+        expect(TslrClaim.new(bank_sort_code: "12 34 56")).to be_valid
+      end
+      it "validates the length of bank_sort_code" do
```

Comment:
Same as above. Also could we have an empty line between specs?

Reply:
👍